backend/data/processors/conditions_preprocessing.py

```diff
@@ -226,6 +233,7 @@ def delete_extraneous_phrasing(processed: str) -> str:
 
     # Remove completion language
     completion_text = [
+        "^currently",
```

Comment:
why do we need `^` here? also will this affect co-requisite handling?

Reply:
Yea, this says that it will remove it only at the beginning of the string. I dont think it hurt any of the coreq handling for this reason